### PR TITLE
Test issue to practice contribution workflow

### LINKED ISSUE #14: Test issue to practice contribution workflow
I'm going to make a small change to try things out!

### PR DESCRIPTION
**Pull request by aktwo**
I added a link to the Google style guide. 

Fixes #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Particula is a simple, fast, and powerful particle simulator, or at least two of
 
 The main goal is to develop an aerosol particle model that is usable, efficient, and productive. In this process, we all will learn developing models in Python and associated packages. Let us all be friendly, respectful, and nice to each other. Any code added to this repository is automatically owned by all. Please speak up if something (even if trivial) bothers you. Talking through things always helps. This is an open source project, so feel free to contribute, however small or big your contribution may be.
 
+We follow the Google Python style guide [here](https://google.github.io/styleguide/pyguide.html).
+
```
Comment (ngam) [inline README.md:10]: Good job on the additional line there (10), otherwise markdown-linter would have annoyed me to fix it!

**Review by ngam — APPROVED** [review]
Welcome aboard, contributor!

**Review by ngam — CHANGES_REQUESTED** [review]
Add "when possible" and "don't let it eat your productivity" please

**Comment by aktwo** [comment]
Whoops, sorry! I was mucking around in vscode and merged without checking. What's interesting is that I would have expected your comment to have blocked my merge, but it looks like you commented on the issue rather than the pull request directly. We can try this again in another commit later!

**Pull request merged**